Resources/views/CRUD/base_history.html.twig

```diff
@@ -33,16 +33,35 @@ file that was distributed with this source code.
                         </thead>
                         <tbody>
                         {% for revision in revisions %}
-                            <tr class="{% if (currentRevision != false and revision.rev == currentRevision.rev) %}current-revision{% endif %}">
+                            <tr class="{{ classlist({
+                                'sonata-ba-revisions__revision': true,
+                                'sonata-ba-revisions__revision--is-current': currentRevision != false and revision.rev == currentRevision.rev
```

Comment:
Exactly :) 80 is very hard with that level of indention, but if you're in an unindented file, try to achieve it.